ui/src/main/java/com/wireguard/android/model/TunnelManager.kt

```diff
@@ -65,6 +66,10 @@ class TunnelManager(private val configStore: ConfigStore) : BaseObservable() {
         // Make sure nothing touches the tunnel.
         if (wasLastUsed)
             lastUsedTunnel = null
+        // Make sure we also remove any existing shortcuts.
+        if (shortcutManager.hasShortcut(tunnel.name)) {
+            shortcutManager.removeShortcuts(tunnel.name)
```

Comment:
Can't we just call remove unconditionally, and it'll be a no-op if they're not already there? What do we gain by first checking?